txscript/engine.go

```diff
@@ -328,6 +328,13 @@ func (vm *Engine) Step() (done bool, err error) {
 		return true, err
 	}
 
+	/*fmt.Println("executing opcode:", opcode.opcode.name)
```

Comment:
NIT: Should either remove this or include a comment on why this is useful to keep. =)